docs/source/noise/index.rst

```diff
@@ -179,8 +179,8 @@ Noise types for each column
     -
   * - SSN
     - W-2 and 1099, SSA
-    - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors
-    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset
+    - Borrow a social security number, leave a field blank, write the wrong digits, make typos, make OCR errors
+    - Note that 'borrow a social security number' only applies to the W-2 and 1099 dataset
```

Comment:
I think you'll have a merge conflict here with https://github.com/ihmeuw/pseudopeople/pull/235 because in that PR you added a period to this line, but hopefully that won't be hard to resolve.